Bottom: 128f1a506433b5924ecc78ee5eb6c6832612264b
Top:    e7442ffb41b7a5013dfd2c0baa5dfba31010be8e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-05-11 15:41:13 +0100

Refresh of add-comment-explaining-assumpt

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 5147492..b325fb5 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2256,8 +2256,8 @@ poly_M_CheckInside (POLYAREA * p, Vector v0)
 
 /* NB: This function assumes the caller _knows_ the contours do not
  *     intersect. If the contours intersect, the result is undefined.
- *     This function will return an incorrect results if the contours
- *     share a commnon node at the arbitrary point tested.
+ *     This function can return an incorrect (positive) result if the
+ *     contours share a common node at the arbitrary point tested.
  */
 int
 poly_ContourInContour (PLINE * poly, PLINE * inner)
